libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/MapboxNotificationData.kt

```diff
@@ -2,4 +2,13 @@ package com.mapbox.navigation.base.trip.model
 
 import android.app.Notification
 
-data class MapboxNotificationData(val notificationId: Int, val notification: Notification)
+/**
+ * Defines data needed for displaying notification and passed to [TripNotification]
+ *
+ * @param notificationId ID of displayed notification
+ * @param notification [Notification] object for displayed notification
+ */
+data class MapboxNotificationData(
```

Comment:
This class is only used in the `core` module. Can it be moved there and made `internal` @olegzil @abhishek1508?

Reply:
Agree with @LukasPaczos This should be moved to `libnavigation-core` module as currently it's only accessed from there.